src/Core/Services/MetadataProviders/MsSqlMetadataProvider.cs

```diff
@@ -40,5 +45,35 @@ public override Type SqlToCLRType(string sqlType)
         {
             return TypeHelper.GetSystemTypeFromSqlDbType(sqlType);
         }
+
+        public override async Task PopulateTriggerMetadataForTable(string schemaName, string tableName, SourceDefinition sourceDefinition)
+        {
+            string queryToGetNumberOfEnabledTriggers = SqlQueryBuilder.GetQueryToGetEnabledTriggers();
```

Comment:
Thanks for the suggestion! I started with the count of triggers, but turns out we need to only use SELECT instead of OUTPUT when a trigger is enabled. Forgot to update the name.